src/main/java/org/apelsin/tutorial/collections/ArrayCollection.java

```diff
@@ -13,4 +15,27 @@ public ArrayCollection(int capacity){
         content=new Object[capacity];
     }
 
+    public Object get(int i){
+        Objects.checkIndex(i, capacity);
+        return content[i];
+    }
+
+    public boolean isEmpty(){
```

Comment:
Too expensive calculation, user global `length` variable instead of local, update global variable when added or deleted something then calculate in `isEmpty` method `this.length == 0`